projects/canopy/src/lib/orientation/docs/orientation.stories.ts

```diff
@@ -0,0 +1,85 @@
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+
+import { LgCardModule } from '../../card';
+import { LgOrientationModule } from '../orientation.module';
+import { LgOrientationDirective } from '../orientation.directive';
+import { LgPaddingModule } from '../../spacing/padding/padding.module';
+
+const responsiveCategory = 'Responsive';
+
+export default {
+  title: 'Helpers/Directives/Orientation/Examples',
+  decorators: [
+    moduleMetadata({
+      imports: [ LgOrientationModule, LgCardModule, LgPaddingModule ],
+    }),
+  ],
+  parameters: {
+    a11y: {
+      disable: true,
+    },
+    viewMode: 'story',
+    previewTabs: { 'storybook/docs/panel': { hidden: true } },
+    backgrounds: {
+      default: 'Super Blue',
+    },
+  },
+  argTypes: {
+    orientation: {
+      name: 'orientation',
+      table: {
+        category: responsiveCategory,
+      },
+    },
+  },
+} as Meta;
+
+const orientationStory: Story<LgOrientationDirective> = (
+  args: LgOrientationDirective,
+) => ({
+  props: args,
+  template: `
+<lg-card
+  lgShadow
+  [lgOrientation]="orientation">
+  <lg-card-hero-img
```

Comment:
This image at 320px doesn't look fully correct:

![image](https://github.com/Legal-and-General/canopy/assets/8397116/b166da29-e35b-4a9f-ab8e-1ec11d11c109)

Reply:
there was a ``min-width`` set on the ``LgCardHeroImageComponent`` and that was conditioning the image width on mobile to look like this on small screens. I got rid of it as it was not necessary. Should be fine now :)